cg/meta/workflow/microsalt/quality_controller/quality_controller.py

```diff
@@ -85,7 +85,7 @@ def quality_control_sample(self, sample_id: str, metrics: SampleMetrics) -> Samp
         )
 
         if has_non_microbial_apptag(sample):
-            sample_passes_qc = True
+            sample_passes_qc = valid_read_count
```

Comment:
Shouldn't this be detected in the tests? does this function have a test?

Reply:
There is no test for samples with non microbial apptags, I'll leave it as is for now